imports/server/api/resources/hunts.ts

```diff
@@ -0,0 +1,113 @@
+import { check, Match } from "meteor/check";
+import { Meteor } from "meteor/meteor";
+import bodyParser from "body-parser";
+import express from "express";
+import type { GdriveMimeTypesType } from "../../../lib/GdriveMimeTypes";
+import GdriveMimeTypes from "../../../lib/GdriveMimeTypes";
+import type { HuntType } from "../../../lib/models/Hunts";
+import Hunts from "../../../lib/models/Hunts";
+import MeteorUsers from "../../../lib/models/MeteorUsers";
+import type { TagType } from "../../../lib/models/Tags";
+import Tags from "../../../lib/models/Tags";
+import addPuzzle from "../../addPuzzle";
+import expressAsyncWrapper from "../../expressAsyncWrapper";
+
+const hunts = express.Router();
+hunts.use(bodyParser.json());
+
+// GET /hunts - list all hunts
+hunts.get(
+  "/",
+  expressAsyncWrapper(async (_, res) => {
+    const userId = Meteor.userId();
+    check(userId, String);
+
+    const user = await MeteorUsers.findOneAsync({ _id: userId });
+    if (!user) {
+      // Should never happen if the API key passed authentication.
+      res.sendStatus(500);
+      return;
+    }
+
+    const allHunts = Hunts.find({}, { sort: { createdAt: -1 } });
```

Comment:
This is on me for not noticing it before, but any reason this can't just be `Hunts.find({ _id: { $in: user.hunts ?? [] }})`?